quantum/nodes/grovers/grovers.js

```diff
@@ -14,7 +14,8 @@ module.exports = function(RED) {
     this.on('input', async function(msg, send, done) {
       let error = errors.validateGroversInput(msg);
       if (error) {
-        done(error);
+        this.error(error.message);
+        done();
```

Comment:
```suggestion
        done(error);
```
For consistency, we should stick to using this method of error handling here.

Reply:
This is a way to deal with #83, do you have alternative ways to test error conditions via test helper?

Reply:
If there are no other way around it, than I suggest for consistency to change all error handling in all nodes like I did here.

Reply:
@louislefevre What do you think about it?

Reply:
I'll look into it this weekend and see if there is a way where we can continue using the `done(error)` format, since using `this.error(error.message)` is the outdated format and isn't ideal.

Reply:
@louislefevre what should we do with this error handling?

Reply:
Do you know what the reason is that prevents `done(error)` from working?